types/googlemaps/index.d.ts

```diff
@@ -381,45 +384,55 @@ declare namespace google.maps {
      * Controls that are added first are positioned closer to the edge of the map.
      */
     enum ControlPosition {
+        /** Same as `BOTTOM_CENTER`. */
+        BOTTOM = 11,
         /** Elements are positioned in the center of the bottom row. */
-        BOTTOM_CENTER,
+        BOTTOM_CENTER = 11,
         /**
          * Elements are positioned in the bottom left and flow towards the middle.
          * Elements are positioned to the right of the Google logo.
          */
-        BOTTOM_LEFT,
+        BOTTOM_LEFT = 10,
         /**
          * Elements are positioned in the bottom right and flow towards the middle.
          * Elements are positioned to the left of the copyrights.
          */
-        BOTTOM_RIGHT,
+        BOTTOM_RIGHT = 12,
+        /** Elements are positioned in the center. */
+        CENTER = 13,
+        /** Same as `LEFT_TOP`. */
+        LEFT = 5,
         /**
          * Elements are positioned on the left, above bottom-left elements, and flow
          * upwards.
          */
-        LEFT_BOTTOM,
+        LEFT_BOTTOM = 6,
         /** Elements are positioned in the center of the left side. */
-        LEFT_CENTER,
+        LEFT_CENTER = 4,
         /**
          * Elements are positioned on the left, below top-left elements, and flow
          * downwards.
          */
-        LEFT_TOP,
+        LEFT_TOP = 5,
+        /** Same as `RIGHT_TOP`. */
+        RIGHT = 7,
         /**
          * Elements are positioned on the right, above bottom-right elements, and
          * flow upwards.
          */
-        RIGHT_BOTTOM,
+        RIGHT_BOTTOM = 9,
         /** Elements are positioned in the center of the right side. */
-        RIGHT_CENTER,
+        RIGHT_CENTER = 8,
         /** Elements are positioned on the right, below top-right elements, and flow downwards. */
-        RIGHT_TOP,
-        /**    Elements are positioned in the center of the top row. */
-        TOP_CENTER,
-        /** Elements are positioned in the top left and flow towards the middle. */
-        TOP_LEFT,
+        RIGHT_TOP = 7,
+        /** Same as `TOP_CENTER`. */
+        TOP = 2,
+        /** Elements are positioned in the center of the top row. */
+        TOP_CENTER = 2,
         /** Elements are positioned in the top right and flow towards the middle. */
-        TOP_RIGHT,
+        TOP_LEFT = 1,
+        /** Elements are positioned in the top right and flow towards the middle. */
+        TOP_RIGHT = 3,
```

Comment:
Please mark options that are not in the [documentation] as `@deprecated`.

![image](https://user-images.githubusercontent.com/10235949/67030092-5559c880-f117-11e9-9f62-cf01d6ea53a2.png)


[documentation]: https://developers.google.com/maps/documentation/javascript/reference/control#ControlPosition

Reply:
Thanks for the comments, I wasn't aware that these were deprecated, so I decided to remove them instead.